data_structures/src/chain.rs

```diff
@@ -1042,15 +1036,72 @@ pub struct ExtendedSecretKey {
     pub chain_code: Protected,
 }
 
-// FIXME: ExtendedSecretKey shouldn't implement Default
-impl Default for ExtendedSecretKey {
-    fn default() -> Self {
+/// BLS data structures
+
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, ProtobufConvert)]
+#[protobuf_convert(pb = "witnet::BlsPublicKey")]
+pub struct BlsPublicKey {
```

Comment:
Can we name it somehow with the curve we are referring to?

Reply:
Something like `Bn256PublicKey`?

Reply:
works